db/pre-re/item_db.conf

```diff
@@ -66942,7 +66952,7 @@ item_db: (
 	Job: 0x00000400
 	Upper: 8
 	BuyingStore: true
-	Script: <" //madoheal rand(300,400),0; ">
+	Script: <" if(checkmount() == MOUNT_MADO) itemheal rand(300,400), 0; ">
```

Comment:
yes, in official, 3rd job never exist in pre-renewal. but in our System, it is enabled but it has limitations.